controllers/apps/cluster_plan_builder.go

```diff
@@ -146,6 +147,7 @@ func init() {
 	model.AddScheme(snapshotv1beta1.AddToScheme)
 	model.AddScheme(extensionsv1alpha1.AddToScheme)
 	model.AddScheme(workloadsv1.AddToScheme)
+	model.AddScheme(parametersv1alpha1.AddToScheme)
```

Comment:
From the perspective of their functionality and relationships, apps should no longer depend on parameters.